src/Microcosm.js

```diff
@@ -226,7 +234,8 @@ Microcosm.prototype = {
    * @param {function} [callback] - A callback to execute after the application starts
    * @return {Microcosm} The invoking instance of Microcosm
    */
-  start(callback) {
+  start(callback=throwIfError) {
+    this.started = true
     install(this.plugins, callback)
     return this
   }
```

Comment:
@nhunzaker If this is anywhere but on the first line, it breaks all the things. I assume this is because the app is calling push internally somewhere during the `install` process? Halp.